Looms.PoS/Looms.PoS.Application/Features/Product/Commands/UpdateProduct/UpdateProductCommandHandler.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Looms.PoS.Application.Interfaces;
+using Looms.PoS.Application.Interfaces.ModelsResolvers;
+using Looms.PoS.Application.Models.Requests.Product;
+using Looms.PoS.Domain.Interfaces;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Looms.PoS.Application.Features.Product.Commands.UpdateProduct;
+
+public class UpdateProductCommandHandler : IRequestHandler<UpdateProductCommand, IActionResult>
+{
+     private readonly IProductsRepository _productsRepository;
+    private readonly IHttpContentResolver _httpContentResolver;
```

Comment:
fix indentation

Reply:
fixed in "fixed indentation/layout problems, changed Variation to Variation request and made create and update requests take IEnumerable<VariationRequest>"